<launch>
	<arg name="config" default="$(find ocean_ros)/config/config.ini" />

    <node ns="ocean" pkg="ocean_ros" type="create_datatoken.py" name="datatoken_creator" respawn="true" output="screen">
        <param name="config" value="$(arg config)" />
    </node>

    <node ns="ocean" pkg="ocean_ros" type="buy_datatoken.py" name="datatoken_downloader" respawn="true" output="screen">
        <param name="config" value="$(arg config)" />
    </node>

    <node ns="ocean" pkg="ocean_ros" type="create_pool.py" name="pool_creator" respawn="true" output="screen">
        <param name="config" value="$(arg config)" />
    </node>
</launch>